### DISCUSSION TITLE
Next.js v13 with /app ignoring root.

### DISCUSSION THREAD

**Discussion by OllieJT** [vercel/next.js#42193] [Help]
I've just spun up a minimal nextjs app and replaced the pages folder with an app folder. Everything runs fine on `yarn dev` it also builds fine with `yarn build` then `yarn start`... however when deployed (to vercel) I get [a 404 page](https://knomii.coach) which is weird - though even more so once I realised [adding /index](https://www.knomii.coach/index) works.

Root URL does not work: `https://knomii.coach`
Root url with /index does work: `https://knomii.coach/index`

**I'm not sure how to debug this further. Is this me? Is this Next? or is this Vercel?**

This is a new project I'm working on. I've enabled the app directory in the next config (reminder that everything is working in dev) - all I have in my app directory is the following:

```
...
src/
    ...
    app/
        head.tsx - this just changes the title
        layout.tsx - this adds a custom font, css file, and a favicon in the head
        page.tsx - this is literally a div with an svg logo.
    styles/
tsconfig.json
package.json
```

You can find the repository here: [OllieJT/knomii-site](https://github.com/OllieJT/knomii-site)

**Comment by OllieJT** [comment]
I have also just tried moving my project out of the `/src` folder in line with the examples in the beta docs. This hasn't changed the issue.

I've also been comparing each file with the demo repo: [app-playground](https://github.com/vercel/app-playground), to which the layout, and next config are almost identical.

**Comment by OllieJT** [comment]
It seems to be similar to [this discussion](https://github.com/vercel/next.js/discussions/41781), however I'm not seeing errors in the console or browser when running in dev mode or building locally or on Vercel.

What is interesting is that I duplicated the homepage and placed it in a folder `/test` - which works just fine... in theory this is using the same layout and page component as the homepage having the issue.

**Comment by williamiommi** [comment]
Hi, 
have you solved it? 
I had the same issue, removing the locales configuration in next.config.js "solved" the issue for me on the vercel build.
Of course it's not a solution for my use case but just to let you know :)

According to this [comment](https://github.com/vercel/next.js/issues/41980#issuecomment-1304645891) from @leerob. They are working on a guide on how to manage localization inside the app directory since the i18n feature it's not supported.

But yeah...probably it's more a bug rather than something related to the i18n feature not being supported. As you said, I can reach the other pages defined inside the app folder even with the locales configured.
